region/crateria/east.json

```diff
@@ -1658,6 +1665,76 @@
                   ]
                 }
               ]
+            },
+            {
+              "id": 17,
+              "strats": [
+                {
+                  "name": "G-mode Morph",
```

Comment:
I'd rather these were written as 1 to 17, 4 to 17, and 5 to 17

Reply:
If doors are freely connected we have been using a central junction to represent them as they can share a gmode strat. (Like in the Parlor). We have been assuming a careful jump is free when doing g-mode, as that is a tech that is low-level and can't ever be "turned off" like wall jumps. I put 4 here with 1,5 to keep them together, but it would be equivalent if they were moved to their doors.

Reply:
All these extra requirements show this is not free movement.  So using 13 as a base common junction doesn't make sense to me.

Reply:
I get what you mean now.  13 can freely reach the doors.  So by trying to find a path into 14 its sensible to want to check from 13.  So to me, I'd want to do it this way.  But I still think it is going to check the door node requirements and come back and that has been wrong in the past.